tests/unit/test_charm.py

```diff
@@ -107,17 +92,9 @@ def test_config_changed_when_no_cors(self):
         assert: it will get to blocked status waiting for it.
         """
         self.add_database_relations()
+        self.harness.update_config({"cors_origin": ""})
```

Comment:
Is this related to the creation of the admin user?

Reply:
As part of this PR we're also removing the mandatory SMTP settings as a requirement to reach active status